driver/conn.go

```diff
@@ -23,17 +23,14 @@ import (
 
 // Conn is a connection to a database.
 type Conn struct {
-	options  Options
+	options  *Options
 	catalog  *catalog
```

Comment:
while you're here, `type catalog` on line 155 would benefit from having a new name